package.json

```diff
@@ -25,7 +25,7 @@
     "test": "jest app/javascript/ bin/ --coverage",
     "test:watch": "jest app/javascript/ bin/ --watch",
     "postcss": "postcss public/assets/*.css -d public/assets 2> postcss_error.log",
-    "e2e": "cypress open",
+    "e2e": "cypress open --config-file cypress.dev.json",
```

Comment:
Allows us to use our development environment configuration when creating/running tests locally.